src/components/AssetDescriptionComponent.jsx

```diff
@@ -25,15 +25,20 @@ const AssetDescriptionComponent = (props) => {
     };
   }
 
+  const specs = props.assetDetail.specs
```

Comment:
Change this to a function. Also, the implementation can be revised to improve readability and comprehension. You don't need the ternary as specs will either be an object or `null`.  I.e.

```js
const getSpecs = specs => {
  if (!specs) {
    return 'This asset has no specifications'
  }
  
  return Object.entries(specs).map(([label, value]) => (
    <p>{`${label.replace(/_/g, ' ')}: ${value}`}</p>
  ));
}

const AssetDescriptionComponent = props = {...}
```
Important notes to remember:
1. Use `replace` to replace the underscore with empty space. Read more about replace [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace)
2. Capitalize the first letter of each label using [text transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform) of `capitalize`
3. Do not show the `id` in the UI
4. Assign relevant units to the asset specs that have units. You can use an object to store the units. Therefore, the above snippet can be updated to:
```js
const UNITS = {
  memory: 'GB',
  processor_speed: 'GHz',
  screen_size: 'inch',
  storage: 'GB'
};

const Specs = specs => {
  if (!specs) {
    return 'This asset has no specifications'
  }
  
  delete specs.id;
  return Object.entries(specs).map(([label, value]) => {
    const specUnit = UNITS[label] || '';
    return <p>{`${label.replace(/_/g, ' ')}: ${value} ${specUnit}`}</p>;
  });
}

const AssetDescriptionComponent = props = {...}
```
Please test if the code works as I haven't tested it.

![image 2018-09-25 at 4 42 20 pm](https://user-images.githubusercontent.com/12892171/46021062-6ee13a00-c0e8-11e8-8b68-07723964287a.png)

![image 2018-09-25 at 4 51 36 pm](https://user-images.githubusercontent.com/12892171/46021071-756fb180-c0e8-11e8-9e9a-10b0a2ad5c4f.png)